### DISCUSSION TITLE
Affix Numbers please

### DISCUSSION THREAD

**Discussion by tyson-dev** [blizzhackers/pickits#15] [Q&A about pickit codes]
From the NIP Guide page 

[prefix] = number of a prefix that should be on the item, this and suffix number can be a bit hard to find
[suffix] = number of a suffix that should be on the item

Yes hard to find! where please? :D

**Comment by AudriusButkevicius** [comment]
https://github.com/blizzhackers/d2data/blob/master/json/magicprefix.json

Etc

**Reply by tyson-dev** [reply]
Awesome Audrius ty,

can anyone tell which number ( 0, nothing, NULL) applies to no affix?

e.g. Would 

"[Name] == SmallCharm && [Quality] == Magic && [Prefix] >= 1 # [MaxHP] >= 20"

pick any 20 life SC that also has any prefix?

**Reply by AudriusButkevicius** [reply]
I think so
